models/template_model.go

```diff
@@ -99,17 +98,17 @@ func (t Tiers) Value() (driver.Value, error) { return jsonValue(t) }
 // Scan implements database/sql.Scanner interface. Should be defined on the pointer.
 func (t *Tiers) Scan(src interface{}) error { return jsonScan(t, src) }
 
-// TemplateParams represent TemplateParam slice.
-type TemplateParams []TemplateParam
+// ParamsDefinitions represent ParamDefinition slice.
+type ParamsDefinitions []ParamDefinition
```

Comment:
What params is this? I think it's better to rename to `TemplateParamsDefinitions`, because it's in models package and it's not obvious until you open the file.

Reply:
Problem is that type now used in two models, in `template` and in `rule`.

Reply:
`AlertingParamsDefinitions`?

Reply:
Yep, it may be. Or may be `ExprParamsDefinitions`? Technically this object describes expression placeholders.

Reply:
Renamed